cluster-autoscaler/core/scaleup/orchestrator/orchestrator.go

```diff
@@ -591,6 +591,7 @@ func (o *ScaleUpOrchestrator) executeScaleUp(
 		o.clusterStateRegistry.RegisterFailedScaleUp(info.Group, metrics.FailedScaleUpReason(string(aerr.Type())), now)
 		return aerr
 	}
+	o.clusterStateRegistry.UpdateScaleDownDelayStatus(info.Group, time.Now())
```

Comment:
I thought of putting the code in `RegisterOrUpdateScaleUp` but ended up creating a separate function because I am not registering or updating scale up as such.